chromium_src/components/permissions/permission_request.cc

```diff
@@ -160,16 +161,15 @@ PermissionRequest::GetDialogAnnotatedMessageText(
 #endif
 
 bool PermissionRequest::SupportsLifetime() const {
-  const RequestType kExcludedTypes[] = {
-    RequestType::kDiskQuota,
-    RequestType::kMultipleDownloads,
+  const RequestType kExcludedTypes[] = {RequestType::kDiskQuota,
+                                        RequestType::kMultipleDownloads,
 #if BUILDFLAG(IS_ANDROID)
-    RequestType::kProtectedMediaIdentifier,
+                                        RequestType::kProtectedMediaIdentifier,
 #else
     RequestType::kRegisterProtocolHandler,
 #endif  // BUILDFLAG(IS_ANDROID)
 #if BUILDFLAG(ENABLE_WIDEVINE)
-    RequestType::kWidevine
+                                        RequestType::kWidevine
```

Comment:
nit: what's happened here with whitespace change?

Reply:
vscode auto formatting, seems to be a good format tho.